### ISSUE TITLE
Add support for new_params

### ISSUE THREAD

**Issue by heaven** [CanCanCommunity/cancancan#551]
Add an ability to assign some params when executing the `new` action. For example, we have a `JobReview` model and the controller looks like this:

```ruby
class Market::JobReviewsController < Market::MarketController
  load_and_authorize_resource :job, class: "Market::Job"
  load_and_authorize_resource class: "Market::JobReview", through: :job
```

A review is possible only once per `reviewer` and `reviewable` associations on the `review` object, we need to set those before authorizing the action. Would be awesome to be able to define the `new_params` method in the controller for this.

**Comment by coorasse** [comment]
Out of scope for the gem, sorry

**Comment by newfylox** [comment]
Is it still out of scope? Is there a reason why not?
